File: JAVA/jogos.java
import java.util.Scanner;

public class jogos {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Quantos jogos foram vendidos?");
        int quantidadeJogos = scanner.nextInt();

        double valorTotal = quantidadeJogos * 19.9;
        double salario = valorTotal * 0.5;
        double bonus = Math.floor(quantidadeJogos / 15) * (valorTotal * 0.08);

        System.out.printf("Venda total: %.2f\nValor do salario: %.2f\nValor do bonus: %.2f\n", valorTotal, salario, bonus);

        scanner.close();
    }
}

File: C/jogos.c

/*
Cada jogo custa R$ 19.90
50% do valor total é o salário

jogosVendidos = 
salario = jogosVendidos * 0.5
bonus = (jogosVendidos / 15) 

*/

#include <stdio.h>
#include <math.h>

int main (){
    int quantidadeJogos;
 
    printf("Quantos jogos foram vendidos?\n");
    scanf("%d", &quantidadeJogos);


    float 
        valorTotal = quantidadeJogos * 19.9, 
        salario = valorTotal * 0.5,
        bonus = floor(quantidadeJogos / 15) * (valorTotal * 0.08);
        

    printf("Venda total: %0.2f \nValor do salario: %0.2f \nValor do bonus: %0.2f\n", valorTotal, salario, bonus);
    
    return 0;
}